ipaserver/plugins/idviews.py

```diff
@@ -94,7 +95,7 @@ class idview(LDAPObject):
     container_dn = api.env.container_views
     object_name = _('ID View')
     object_name_plural = _('ID Views')
-    object_class = ['ipaIDView', 'top']
+    object_class = ['ipaIDView', 'ipaNameResolutionData', 'top']
```

Comment:
IMO `ipaNameResolutionData` must be in `possible_object_class`,  because idviews after upgrade doesn't have it